oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/HistoryDeleteEsDAO.java

```diff
@@ -63,20 +62,26 @@ public void deleteHistory(Model model, String timeBucketColumnName, int ttl) {
             }
             return;
         }
+
+        String latestIndex = TimeSeriesUtils.latestWriteIndexName(model);
+        if (!client.isExistsIndex(latestIndex)) {
+            client.createIndex(latestIndex);
+            if (log.isDebugEnabled()) {
+                log.debug("Latest index = {} is not exist, create.", latestIndex);
+            }
+        }
```

Comment:
Please revert the `try-catch` back, the index might be created by by ES when the data is received and if you remove this `try-catch` exception might be thrown.

Even if you check `if (!client.isExistsIndex(latestIndex)) {` but data might be sent to ES after the `if` statement before the `client.createIndex(latestIndex);`

Reply:
>  the index might be created by ES when the data is received

Although this should never happen when the TTL job is working fine, as it will create the index for the next day in advance, this a scenario to take into consideration.